.gitignore

```diff
@@ -8,3 +8,7 @@ spec/dummy/log/*.log
 spec/dummy/public/assets
 spec/dummy/tmp/
 spec/dummy/.sass-cache
+
+# bower-rails
```

Comment:
I don't think so. `bower.json` is generated from the `Bowerfile`. `.bowerrc` is generated if it's not there so it's configured to look in the right directory for the assets. We can change `.bowerrc` and it won't be overwritten. `bower.json` will be.